modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1139,6 +1138,21 @@ export class Wallet implements IWallet {
       addressParams.format = format;
     }
 
+    if (this.baseCoin.getFamily() === 'ofc') {
+      if (!_.isUndefined(onToken)) {
+        if (!_.isString(onToken)) {
+          throw new Error('onToken has to be a string');
+        }
+        if (!ofcTokens.includes(onToken)) {
+          console.log('ofcTokens', ofcTokens);
+          throw new Error('Unknown OFC token');
+        }
+        addressParams.onToken = onToken;
+      } else {
+        throw new Error('onToken is a mandatory parameter for OFC wallets');
+      }
```

Comment:
Nit: can simply this check

```suggestion
if((!_.isString(onToken)) {
	throw new Error('onToken is a required string');
}
if (!ofcTokens.includes(onToken)) {
	console.log('ofcTokens', ofcTokens);
	throw new Error('Unknown OFC token');
}
	addressParams.onToken = onToken;
```